doc/faq.md

```diff
@@ -11,10 +11,6 @@ description: none
 Riot is against the current trend of boilerplate and unneeded complexity. We think that a small, powerful API and concise syntax are extremely important things on a client-side library.
 
 
-### Why IE8?
-Because it's widely used. According to [Net Market Share](http://www.netmarketshare.com/) the global desktop market share is 19% and according to [StatCounter](http://statcounter.com/demo/browser/) the market share is 2.6%.
-So Please consider [IE8 support](https://muut.com/riotjs/download.html#ie8-support) on your project.
```

Comment:
If IE8 support is being fully dropped we should clean up the [linked segment on download page](https://muut.com/riotjs/download.html#ie8-support) - or re-add this line with a note that IE8 is not actually supported and will require more work (shims, coding style modifications, etc) or whatever is needed